docs/components/multiselect.md

```diff
@@ -36,7 +42,10 @@ You may also specify:
     selected: true
   }, {
     label: 'Dogs',
-    value: 'dogs'
+    value: 'dogs',
+    selected: true,
+    disabled: true,
+    disabledTooltipText: 'This item is non-dismissible'
```

Comment:
Can you change all the instances of `This item is non-dismissible` to `This item is locked and can not be removed`
```suggestion
    disabledTooltipText: 'This item is locked and can not be removed'
```